oap-server/server-storage-plugin/storage-jdbc-hikaricp-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/jdbc/h2/dao/H2NetworkAddressInventoryCacheDAO.java

```diff
@@ -63,7 +63,7 @@ public H2NetworkAddressInventoryCacheDAO(JDBCHikariCPClient h2Client) {
                 try (ResultSet resultSet = h2Client.executeQuery(connection, sql.toString(), lastUpdateTime)) {
                     NetworkAddressInventory addressInventory;
                     do {
-                        addressInventory = (NetworkAddressInventory)toStorageData(resultSet, NetworkAddressInventory.INDEX_NAME, new ServiceInventory.Builder());
+                        addressInventory = (NetworkAddressInventory)toStorageData(resultSet, NetworkAddressInventory.INDEX_NAME, new NetworkAddressInventory.Builder());
```

Comment:
This bug should be merged quickly. Could you send another PR to fix?

Reply:
> This bug should be merged quickly. Could you send another PR to fix?

Yes, I'll send a fix PR soon